pkg/blockstorage/vmware/vmware.go

```diff
@@ -51,11 +51,13 @@ const (
 	// Note: it is up to the creator of the provider to determine if a para-virtualized environment exists.
 	VSphereIsParaVirtualizedKey = "VSphereIsParaVirtualizedKey"
 
-	noDescription     = ""
 	defaultWaitTime   = 60 * time.Minute
 	defaultRetryLimit = 30 * time.Minute
 
 	vmWareTimeoutMinEnv = "VMWARE_GOM_TIMEOUT_MIN"
+
+	DescriptionTag = "kanister-8d6b8dc9ceb74bb4b47b3c0a408658c4"
+	VolumeIdTag    = "kanister-volume-8d6b8dc9ceb74bb4b47b3c0a408658c4"
```

Comment:
I tried to avoid possibility to overlap with user defined variables. Generated GUID seemed to be the best way. I think we should never change them so user could rely on those const when use kanister

Reply:
Generated GUID sounds good to me. Are these constants also shared with downstream? Just want to make sure they don't go out-of-sync, if used in different projects.

Reply:
I don't think you need to get paranoid over this as long as the string is distinct enough. However, easier on the eye than a GUID would be a pseudo-DNS like name: e.g. `kanisterio.kanister/description` and `kanisterio.kanister/volume-id`

Reply:
BTW,  from the code it appears that `VolumeIdTag` should be `VolumeIdListTag`.

Reply:
Mainly because I want to understand the scope and side-effect these values have on downstream. Hard to tell at the moment without comments in the code, or searching through PRs in other projects. Hope the question is reasonable and not overly paranoia.

Reply:
Yes, this const should be use from other projects, so they always will be synchronised even if we change the name in future(but if we change it, snapshots created in older version becomes inaccessible) 
@carlbraganza if you think that dns-like naming is enough I'll change this

Reply:
Yes, DNS like naming is sufficiently unique.

Reply:
> but if we change it, snapshots created in older version becomes inaccessible

Thanks for the clarification. I'd think this is worth mentioning at least mention in a code comment, to cautious others who may stumble across this code in the future, to avoid breaking other projects.